src/3p-frame.js

```diff
@@ -75,7 +72,7 @@ function getFrameAttributes(parentWindow, element, opt_type) {
     referrer: viewer.getUnconfirmedReferrerUrl(),
     canonicalUrl: docInfo.canonicalUrl,
     pageViewId: docInfo.pageViewId,
-    clientId: element.getAttribute('ampcid'),
+    clientId: null,
```

Comment:
we have other element use `getIframe()` function. Before they will always get a clientId: null. 
I am trying to keep behavior constant here. I don't know if they check `global.context.clientId` in their code.